### PR TITLE
java-deobfuscator 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by yumitsu**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/java-deobfuscator.rb
Comment (vinyldarkscratch) [inline Formula/java-deobfuscator.rb]: Hey there, this line is preventing the test bot from completing what it needs to.  Can you write/include a test for this formula?

**Review by queengooborg — CHANGES_REQUESTED** [review]

File: Formula/java-deobfuscator.rb
Comment (yumitsu) [reply Formula/java-deobfuscator.rb]: I can if you dont mind that `test` block will be really huge in comparison with `install` block :)

Comment (vinyldarkscratch) [reply Formula/java-deobfuscator.rb]: Not at all, thank you very much!  😄

Comment (yumitsu) [reply Formula/java-deobfuscator.rb]: @vinyldarkscratch Well, definitely not for now - during writing the 'testcase' I found that `java-deobfuscator` uses `sunmscapi.jar`, which is Windows dependent external lib.
I opened an issue, let's see what will happen.

Comment (vinyldarkscratch) [reply Formula/java-deobfuscator.rb]: *!  Since all new formulas require tests, we’ll have to hold off before we’ll be able to get this merged, sadly.

Comment (yumitsu) [reply Formula/java-deobfuscator.rb]: @vinyldarkscratch No problem, maintainer said he fixed this issue, I will check it soon.

**Comment by ilovezfs** [comment]
Sorry but we can't accept pull requests that have `false` for their test block.

**Pull request closed without merging**
